config/scripts/validate-fonts.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+yarn build-fonts > /dev/null
+
+if (git diff --quiet src/generated/material-symbols-rounded-optimized.woff2); then
+  echo "Optimized material-symbols font is up to date."
+else
+  echo "Optimized material-symbols font is out of date."
+  echo "Please run 'yarn build-fonts' to update src/generated/material-symbols-rounded-optimized.woff2."
```

Comment:
but we just ran `yarn build-fonts`, didn't we? Shouldn't the message say "please commit the updated font" or something?

Reply:
In most cases, we will see this in the CI since the validate-fonts task is run as part of the validate-ci task.
So the message will make total sense there, and indeed if you run it yourself locally you'll have one step that's not needed in the message. But I don't think it will happen because if you think of running the check task, you probably run the build command directly instead, and then there is no need for the check task.

Reply:
Gotcha, thanks.